documentation/src/main/asciidoc/topics/proc_query_using_analyzers.adoc

```diff
@@ -0,0 +1,21 @@
+[id='using-analyzers_{context}']
+= Using analyzers
+Analyzers convert input data into terms that you can index and query.
+You specify analyzer definitions with the `@Field` annotation in Protobuf schema or Java classes.
+
+.Procedure
+
+. Include the `Analyze.YES` attribute to indicate that the property is analyzed.
+. Specify the analyzer definition with the `@Analyzer` annotation.
+
+.Protobuf schema
+[source,protobuf,options="nowrap",role="primary"]
+----
+include::protobuf/testentity.proto[]
+----
+
+.Java classes
+[source,java,options="nowrap",role="secondary"]
+----
+include::code_examples/AnalyzerDefinitions.java[]
+----
```

Comment:
This section probably need to be split into "embedded" and "remote".  The description above is applicable for Java Hot Rod client only.

Embedded query is completely different and uses Hibernate Search annotations. E.g.:

```java
@FullTextField(analyzer = "...")
public String getNotes() {
      return notes;
   }
```

Valid values for analyzer are https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#backend-lucene-analysis-builtin.

We have very little tests for ```@FullTextField``` + analyzers  and are still using the deprected ```@Analyzer``` annotation in the test suite. 

Suggestion: don't mention about embedded and analyzers for now and mark what we have in this PR as "remote query" only

Reply:
Sounds good. I've moved the analyzers back to the assembly for remote caches. I agree that it does add complexity to mix the Hibernate Search with ProtoStream annotations on top of remote and embedded. The Hibernate Search docs are very good so we can just leave embedded users to it. Thanks again.